From: SeongJae Park <sj@kernel.org>
Date: Mon, 17 Nov 2025 18:23:33 -0800
Subject: [PATCH] mm/vmalloc: temporal build error fix

The bug is reported, so should be fixed soon.  Add the fix temporarily,
though.

https://lkml.kernel.org/r/20251118-only-one-vmalloc-v1-1-29f8eacb4605@ixit.cz

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 mm/vmalloc.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mm/vmalloc.c b/mm/vmalloc.c
index d78c4d09d6ea..0469ba2c230e 100644
--- a/mm/vmalloc.c
+++ b/mm/vmalloc.c
@@ -3936,7 +3936,7 @@ static gfp_t vmalloc_fix_flags(gfp_t flags)
 	gfp_t invalid_mask = flags & ~GFP_VMALLOC_SUPPORTED;
 
 	flags &= GFP_VMALLOC_SUPPORTED;
-	WARN_ONCE(1, "Unexpected gfp: %#x (%pGg). Fixing up to gfp: %#x (%pGg). Fix your code!\n"
+	WARN_ONCE(1, "Unexpected gfp: %#x (%pGg). Fixing up to gfp: %#x (%pGg). Fix your code!\n",
 		  invalid_mask, &invalid_mask, flags, &flags);
 	return flags;
 }
-- 
2.47.3

